vochain/scrutinizer/vote.go

```diff
@@ -87,7 +87,7 @@ func (s *Scrutinizer) GetEnvelope(nullifier []byte) (*indexertypes.EnvelopePacka
 		return nil, fmt.Errorf("transaction is not an Envelope")
 	}
 	log.Debugf("getEnvelope took %s", time.Since(t))
-	return &indexertypes.EnvelopePackage{
+	envPkg := &indexertypes.EnvelopePackage{
```

Comment:
`envPkg` is not very self descriptive, I'd name it just envelope or envelopePkg (or envelopePackage)